### DISCUSSION TITLE
Generous free tier for students

### DISCUSSION THREAD

**Discussion by immanuel-peter** [supabase/supabase#29381] [Feature Requests]
Can Supabase allow Pro plans to be at a discount or free through GitHub Student Developer Pack, please? It's really hard for students who want to create working prototypes of projects to demonstrate them if the pricing of developer necessities is out of reach.

**Comment by NiceSapien** [comment]
fr, appwrite has free github education plan. I tried it, it was great - except for the SDK, which, for the flutter version was using outdated dependencies disabling me from building my app entirely! So I started looking for supabase and here we are...

**Comment by deraljoscha** [comment]
push

**Comment by taozhe6** [comment]
Hi everyone,

I'm adding my voice to this discussion and wanted to provide some important context based on my research.

I'm also a university student and recently reached out to Supabase support about a student program. My inquiry was based on a specific, historical policy mentioned by the Supabase team.

In a 2021 GitHub Discussion (Questions about pricing #1173), co-founder kiwicopple and maintainer roryw10 explicitly confirmed that "University (...) Students will be eligible for 2 years of base tier usage" (which is now the Pro tier).

This suggests that a generous student program was not just a request, but an actual policy at one point.

My support ticket regarding this is SU-231840, but I was directed to the discussions for follow-up.

Could someone from the Supabase team perhaps clarify the current status of this program?

Is the offer for 2 years of Pro tier credits still available to university students?
If so, what is the correct application process in 2025?
If the policy has changed, could you share the current official stance on student benefits?

Clarification would be incredibly helpful for all the students here hoping to use Supabase for their learning and projects. Thank you!

**Reply by GaryAustin1** [reply]
I’ve been helping them out for more than 3 years and have not seen a Student plan offered.

**Reply by taozhe6** [reply]
as a volunteer?, you are so nice!

**Reply by GaryAustin1** [reply]
> as a volunteer?, you are so nice!

Started that way. They offered to comp for time after awhile.

**Reply by Adrou** [reply]
This?
https://supabase.com/blog/supabase-striveschool

**Reply by GaryAustin1** [reply]
That link is from 2020.  I've never seen that mentioned since then.

**Reply by taozhe6** [reply]
> This?
> 
> https://supabase.com/blog/supabase-striveschool

this is for chairman of program school
